addons/project/report/project_report.py

```diff
@@ -43,6 +43,7 @@ class ReportProjectTaskUser(models.Model):
         ('1_canceled', 'Cancelled'),
         ('02_changes_requested', 'Changes Requested'),
     ], string='State', readonly=True)
+    is_closed = fields.Boolean(string='Closed state', readonly=True)
```

Comment:
We should compute it in the SQL query to be able to use it. 🙂 
Otherwise, its value will be always False